lib/src/chip/chip.component.scss

```diff
@@ -0,0 +1,77 @@
+@import '../sass/cashmere';
+
+.chip {
+    border-radius: 6px;
+    height: 32px;
+    line-height: 32px;
+    @include fontSize(14px);
+    display: inline-block;
+    padding: 0 15px;
+    cursor: pointer;
+    margin: 0 5px 10px 5px;
+}
+
+.neutral {
+    color: $text;
+    background-color: tint( $text, 87%);
+}
+
+.red {
+    color: $ruby-red;
+    background-color: tint( $ruby-red, 85%);
+    border: 2px solid tint( $ruby-red, 75%);
+    line-height: 28px !important;
+}
+
+.yellow {
+    color: shade( $yellow-orange, 25%);
+    background-color: tint( $yellow-orange, 80%);
+    border: 2px solid tint( $yellow-orange, 50%);
+    line-height: 28px !important;
+}
+
+.green {
+    color: $green;
+    background-color: tint( $light-green, 60%);
+    border: 2px solid tint( $light-green, 30%);
+    line-height: 28px !important;
+}
+
+
+.close:after {
+    content:"\f00d";
+    font-family: FontAwesome;
+    @include fontSize(15px);
+    padding-left: 12px;
+    opacity: 0.4;
+}
+
+:host(.filter-button) {
+    font-weight: 600;
+    display: inline-block;
+    @include fontSize(16px);
+    width: 120px;
+    line-height: 32px;
+    color: $primary-brand;
+    cursor: pointer;
+}
+
+.filter-img {
+    height: 27px;
+    width: 27px;
+    padding: 2px 8px 4px 0;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+:host(.chip-row) { display: flex; }
+
+.chip-row-button { display: inline-block; }
+
+.chip-row-contents { display: inline-block; }
+
+.single-row {
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+}
```

Comment:
I just want to congratulate you. These styles are nicely defined. Too often people make css definitions extremely specific